File: backend/src/main/java/com/tripfriend/domain/member/member/dto/MemberDto.java
package com.tripfriend.domain.member.member.dto;

import com.tripfriend.domain.member.member.entity.Member;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class MemberDto {
    private Long id;
    private String username;

    public MemberDto(Member member) {
        this.id = member.getId();
        this.username = member.getUsername();
    }
}

File: backend/src/main/java/com/tripfriend/domain/member/member/dto/MemberDto.kt
package com.tripfriend.domain.member.member.dto

import com.tripfriend.domain.member.member.entity.Member

data class MemberDto(
    val id: Long?,
    val username: String
) {
    constructor(member: Member) : this(
        id = member.id,
        username = member.username
    )
}
